Crash on Background Task

************ CAUSE OF ERROR ************

java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$4.done(AsyncTask.java:415)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
	at java.util.concurrent.FutureTask.run(FutureTask.java:271)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:923)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Context.getPackageName()' on a null object reference
	at com.owncloud.android.operations.RefreshFolderOperation.sendLocalBroadcast(RefreshFolderOperation.java:733)
	at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:255)
	at com.owncloud.android.lib.common.operations.RemoteOperation.execute(RemoteOperation.java:143)
	at com.owncloud.android.ui.fragment.contactsbackup.ContactsBackupFragment$2.doInBackground(ContactsBackupFragment.java:235)
	at com.owncloud.android.ui.fragment.contactsbackup.ContactsBackupFragment$2.doInBackground(ContactsBackupFragment.java:226)
	at android.os.AsyncTask$3.call(AsyncTask.java:394)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	... 4 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30170090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: google
Device: blueline
Model: Pixel 3
Id: RQ3A.210905.001
Product: blueline

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: 7511028

--- a/src/main/java/com/owncloud/android/ui/fragment/contactsbackup/ContactsBackupFragment.java
+++ b/src/main/java/com/owncloud/android/ui/fragment/contactsbackup/ContactsBackupFragment.java
@@ -198,7 +198,9 @@ public class ContactsBackupFragment extends FileFragment implements DatePickerDi
         final ContactsPreferenceActivity contactsPreferenceActivity = (ContactsPreferenceActivity) getActivity();
         if (contactsPreferenceActivity != null) {
             String backupFolderPath = getResources().getString(R.string.contacts_backup_folder) + OCFile.PATH_SEPARATOR;
-            refreshBackupFolder(backupFolderPath, contactsPreferenceActivity, contactsPreferenceActivity.getStorageManager());
+            refreshBackupFolder(backupFolderPath,
+                                contactsPreferenceActivity.getApplicationContext(),
+                                contactsPreferenceActivity.getStorageManager());
         }
     }
 
@@ -212,7 +214,7 @@ public class ContactsBackupFragment extends FileFragment implements DatePickerDi
 
                 if (folder != null) {
                     RefreshFolderOperation operation = new RefreshFolderOperation(folder, System.currentTimeMillis(),
-                            false, false, storageManager, user, getContext());
+                            false, false, storageManager, user, context);
 
                     RemoteOperationResult result = operation.execute(user.toPlatformAccount(), context);
                     return result.isSuccess();

